<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>90</y>
     <width>62</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_passport">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>130</y>
     <width>62</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>130</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="server">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>162.105.81.242</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_server">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoLogin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>自动登陆</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoRememberPwd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>97</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
